src/player_hardcoded_effects.cpp

```diff
@@ -885,6 +898,270 @@ void player::hardcoded_effects( effect &it )
             g->events().send<event_type::dies_from_drug_overdose>( getID(), id );
             hp_cur[hp_torso] = 0;
         }
+    } else if( id == effect_hypovolemia ) {
+        // bleed out lambda
+        auto bleed_out = [ this ] {
+            if( has_effect( effect_bleed ) )
+            {
+                add_msg_player_or_npc( m_bad,
+                                       _( "You bleed to death!" ),
+                                       _( "<npcname> bleeds to death!" ) );
+                g->events().send<event_type::dies_from_bleeding>( getID() );
+            } else
+            {
```

Comment:
```suggestion
            } else {
```
pretty sure you'ore gonna need to atyle